### PR TITLE
[FIX] Prevent call button toggle when user is on call

### PR DESCRIPTION
**Pull request by KevLehman**

### CODE REVIEW

File: client/sidebar/sections/components/OmnichannelCallToggleReady.tsx [hunk 1]
```diff
@@ -28,6 +28,13 @@ export const OmnichannelCallToggleReady = (): ReactElement => {
 	}, [voipClient]);
 	// TODO: move registration flow to context provider
 	const handleVoipCallStatusChange = useMutableCallback((): void => {
+		const { state } = voipClient.callerInfo;
+
+		if (['IN_CALL', 'OFFER_RECEIVED', 'ON_HOLD'].includes(state)) {
```
Comment (MartinSchoeler) [inline client/sidebar/sections/components/OmnichannelCallToggleReady.tsx:33]: pls remove the 'offer received' one from the list

Comment (MartinSchoeler) [inline client/sidebar/sections/components/OmnichannelCallToggleReady.tsx:33]: pls remove the 'offer received' one from the list

File: definition/voip/VoIpCallerInfo.ts [hunk 2]
```diff
@@ -13,17 +13,7 @@ export type VoIpCallerInfo =
...
-			caller: ICallerInfo;
-	  }
-	| {
-			state: 'IN_CALL';
-			userState: UserState;
-			caller: ICallerInfo;
-	  }
-	| {
-			state: 'ON_HOLD';
+			state: 'IN_CALL' | 'OFFER_RECEIVED' | 'ON_HOLD';
```
Comment (MartinSchoeler) [inline definition/voip/VoIpCallerInfo.ts:16]: same here

Comment (MartinSchoeler) [inline definition/voip/VoIpCallerInfo.ts:16]: same here

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]

File: client/sidebar/sections/components/OmnichannelCallToggleReady.tsx (hunk 1 above)
Comment (KevLehman) [reply client/sidebar/sections/components/OmnichannelCallToggleReady.tsx]: image

**Review by MartinSchoeler — DISMISSED** [review]

**Review by MartinSchoeler — APPROVED** [review]

**Pull request merged**